railib/api.py

```diff
@@ -160,9 +158,26 @@ def _get_resource(ctx: Context, path: str, key=None, **kwargs) -> dict:
 # Retrieve a generic collection of resources.
 def _list_collection(ctx, path: str, key=None, **kwargs):
     rsp = rest.get(ctx, _mkurl(ctx, path), **kwargs)
-    rsp = json.loads(rsp)
+    rsp = json.loads(rsp.read())
     return rsp[key] if key else rsp
 
+#
+# Finds and returns the multipart form-data boundary
+# Boundary is something like,
+# Content-Type: multipart/form-data; boundary=d6d47cadd395db7f6648a2ffb65751eb
+#
+
+
```

Comment:
Drop empty lines?

Reply:
I guess it is a convention ... my IDE was showing me lint issues if I was not adding extra two lines

Reply:
I think it's a PEP convention for function signatures/bodies. Weird to apply to doc strings.

Reply:
I think if you remove the first empty `# ` line it'll stop showing that lint issue ;)